src/test/app/forms/validation/validators/minNumberOfPopoulatedRows.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from 'chai'
+
+import { MinNumberOfPopulatedRowsConstraint } from 'forms/validation/validators/minNumberOfPopulatedRows'
+
+const EMPTY: boolean = true
+const POPULATED: boolean = false
+
+describe('MinNumberOfPopulatedRowsConstraint', () => {
+  const constraint: MinNumberOfPopulatedRowsConstraint = new MinNumberOfPopulatedRowsConstraint()
+
+  describe('validate', () => {
+
+    context('should accept', () => {
+
+      it('should accept null value', () => {
```

Comment:
please don't test against null

Reply:
You mean `undefined` or removing this test at all?

Reply:
use `undefined` if the test is valid